pkg/wallet/wallet_create_action_test.go

```diff
@@ -255,16 +257,159 @@ func (s *WalletTestSuite) TestWalletCreateActionSuccess() {
 	})
 }
 
-func (s *WalletTestSuite) TestWalletCreateActionError() {
-	s.Run("return error when user have not enough funds", func() {
+func (s *WalletTestSuite) TestWalletCreateActionNewWithSend() {
+	s.Run("create new action", func() {
 		t := s.T()
+		const topUpValue = testValueForFunding
+
 		// given:
-		given := testabilities.Given(t)
+		given, cleanup := testabilities.Given(t)
+		defer cleanup()
 
 		// and:
-		aliceWallet, cleanup := given.AliceWalletWithStorage(s.StorageType)
+		aliceWallet := given.AliceWalletWithStorage(s.StorageType)
+
+		// and:
+		txFromFaucet, _ := given.Faucet(aliceWallet).TopUp(topUpValue)
+
+		// when:
+		args := fixtures.DefaultWalletCreateActionArgs(t)
+
+		result, err := aliceWallet.CreateAction(t.Context(), args, fixtures.DefaultOriginator)
+
+		// then:
+		assert.NoError(t, err)
+
+		// and:
+		require.NotNil(t, result, "Wallet should return result")
+
+		// and:
+		assert.NotEmpty(t, result.Txid, "Wallet result should have transaction id")
+		assert.NotEmpty(t, result.Tx, "Wallet result should have transaction bytes")
+		assert.Len(t, result.SendWithResults, 1, "Wallet result should have single send with results")
+		assert.Equal(t, result.SendWithResults[0].Txid, result.Txid, "Wallet result should have same txid as the one from send with result")
+		assert.Equal(t, result.SendWithResults[0].Status, sdk.ActionResultStatusUnproven, "Wallet send with result should have unproven status")
+
+		// and check the state of wallet:
+		thenState := testabilities.ThenWalletState(t, aliceWallet)
+		thenState.
+			HasActionsCount(2).
+			HasActionsCount(1, fixtures.CreateActionTestLabel)
+
+		thenState.ActionAtIndex(0).
+			WithTxID(txFromFaucet.ID()).
+			WithSatoshis(topUpValue)
+
+		const fee = 2
+		thenCreatedAction := thenState.ActionAtIndex(1)
+		thenCreatedAction.
+			WithTxID(result.Txid.String()).
+			WithStatus(sdk.ActionStatusUnproven).
+			WithDescription(args.Description).
+			WithLabels(fixtures.CreateActionTestLabel).
+			WithSatoshis(-int64(args.Outputs[0].Satoshis) - fee) // Pay attention that this is negative value (user spends balance).
+
+		thenCreatedAction.OutputAtIndex(0).
+			WithSatoshis(args.Outputs[0].Satoshis).
```

Comment:
It would be good to check the listoutputs also:
```suggestion
			ListActionsAlignsListOutputs().
			WithSatoshis(args.Outputs[0].Satoshis).
```